example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -590,7 +590,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
**Suggestion:** The IPHONEOS_DEPLOYMENT_TARGET is set to 9.0. Consider setting it to a higher version to take advantage of the latest iOS features and improvements.
```suggestion
    IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```